### ISSUE TITLE
Minor discrepancy between INPFC and NMFS boundaries (Unimak Pass) 

### ISSUE THREAD

**Issue by zoyafuso-NOAA** [afsc-gap-products/StationAllocationAIGOA#25]
### Issue

Hi @Ned-Laman-NOAA. In the image below is the historical strata in black and the red boundary is the NMFS statistical boundary area for 610. The historical strata 10 and 11 in Unimak Pass cut into neighboring NMFS statistical area 519 (not shown). 

### Question
Should the new strata have the same footprint (northern extent) as historical strata 10 or 11 or be truncated by area 610? Currently I am using the NMFS boundaries to define the northern extent of the strata and thus preventing bleeding into area 519. I think this also happens in the SE but just wondering how consistent we should be about these NMFS boundaries. I understand that it is frustrating to continue to handle both INPFC and NMFS boundaries even though the former is outdated. 

![image](https://github.com/afsc-gap-products/StationAllocationAIGOA/assets/60302777/de5be97f-9faf-4320-8455-b98bcbec498f)

**Comment by zoyafuso-NOAA** [comment]
The decision after our meeting is to create a separate strata that historically included the area in NMFS are 519. This strata will be used in the regional GOA estimate but not in the calculations across NMFS areas.

**Comment by Ned-Laman-NOAA** [comment]
I recall that Stan also recommended we bring this issue to the Plan Team to ask if they can redefine the southern boundary of 519 further north to match our historic sampling area bounds for the Gulf and Aleutians. Not sure if this would be well received or not, but it's out there now.

**Comment by zoyafuso-NOAA** [comment]
Just wanted to add more information here. So the legacy GOA footprint actually intersects with more NMFS areas than 519 but a lot of these overlaps are really small. The table below shows the area overlap of each NMFS area with the legacy GOA survey footprint.  There seems to be even more overlap with the Southeast Inside District NMFS area 659. I think these two NMFS areas are too small to try to optimize depth strata so I am proposing to keep these areas as one stratum. You’ll see in the images below that the 519 area is fairly shallow but does go down to about 200 m. You’ll also see in the image for the 659 area that a small portion of a deep gully is clipped in the area. And in both instances, there are good tow paths. 

| NMFS_AREA | Area Overlap (km2) | Notes and Action Item                         |
|-----------|--------------------|-----------------------------------------------|
| 509       | 26.72              | Too small, removed                            |
| 517       | 13.38              | Too small, removed                            |
| 518       | 2.43               | Too small, removed                            |
| 519       | 859.75             | Unimak Pass, 1 stratum (519010)               |
| 541       | 2.76               | Too small, removed                            |
| 610       | 63385.17           | Incorporated                                  |
| 620       | 70517.99           | Incorporated                                  |
| 630       | 99782.54           | Incorporated                                  |
| 640       | 36950.49           | Incorporated                                  |
| 649       | 138.76             | Too small, removed                            |
| 650       | 44723.6            | Incorporated                                  |
| 659       | 2715.35            | Southeast Inside District, 1 stratum (659010) |

![image](https://github.com/afsc-gap-products/StationAllocationAIGOA/assets/60302777/3d631be6-f9bc-40f5-847a-8d6d42e65c2f)

![image](https://github.com/afsc-gap-products/StationAllocationAIGOA/assets/60302777/dc285833-e035-40ba-a60b-39faf13abcb4)

**Comment by zoyafuso-NOAA** [comment]
Ultimately, this results in:
-	Two extra strata outside those in NMFS areas 610, 620, 630, 640, 650 that contribute to the biomass estimates aggregated at the region-level. I am not sure how to label these new strata yet, ideas are welcome. 
-	The stratum consisting of NMFS area 519 will contribute to the biomass estimates aggregated at the Western GOA level
-	The stratum consisting of NMFS area 659 will contribute to the biomass estimates aggregated at the Eastern GOA level. 
-	Currently we are also providing biomass estimates aggregated to the NMFS areas from 2025-on. I am not recommending extending these estimates to the 519 and 659 NMFS areas simply because the overlap with legacy GOA footprint only covers a small portion of the 0-1000 m habitat in the NMFS areas. 

What are your thoughts?

**Comment by Ned-Laman-NOAA** [comment]
My thoughts are several.

1. We will have to present this to the Plan Team in September and get their approval for whatever moves we make regarding altering the GOA footprint or addressing management area questions. My strategy would be to do that by providing them with 2 options to consider.
2. Option 1 - we drop all of the extraneous strata but we provide evidence why it won't be a problem if we do (e.g., biomass contributions from those areas are minimal compared to subareas and regional totals, if true).
3. Option 2 - we combine all of the "outside of the main management zone" strata into a single stratum and allocate stations randomly to that area on an annual basis. We would also have to come up with justification to communicate to the Plan Team for this option as well.

What do you think?

**Comment by zoyafuso-NOAA** [comment]
Yeah, let's plan to talk to the Plan Team about this, how do we set that up, when do they need to be notified, and who do we notify? Option 1 might work out for the smaller 519 overlap because it's a very small area, smaller than even the smallest of the historical GOA strata. Option 2 might work out for the SE overlap, I can see arguments for keeping the area in the SE because the species comp is so different out there compared to the rest of the GOA.

**Comment by Ned-Laman-NOAA** [comment]
We have a couple of options for connecting with the Plan Team.  If we have relationships with Plan Team members we can begin socializing the idea once we've settled on an approach (i.e., we could have informal conversations this summer). For the Joint Plan Team meeting in September, they will put out a call ahead of time and we'll respond to get on the agenda.
I'd like to at least start our conversation as an either/or approach to Options 1 & 2 rather than a hybrid option. I got the impression from Stan that his preference is one or the other, not both. I also think that trying to inform the Plan Team and get a decision in September requires that we present a simple and clear set of options and make them pick one.

**Comment by Ned-Laman-NOAA** [comment]
Having reached out to the Plan Team in Fall 2024, I'm working on **getting us on to the November Plan Team Agenda (typically mid-November).** They are asking how much time we'll need to present and discuss our issue. I'd like to keep it brief and see if we can fit it into **a 15-minute time block for presentation and discussion**.
I think we will need to make our case for Option 1 or Option 2 with data. 
For Option 1, we'd need to demonstrate the impact to biomass estimates of dropping historic strata that are now outside of the newly restratified survey area.
For Option 2, we'd want to demonstrate how we would operationalize this strategy and what impacts it would have to biomass.
In both cases, we'll need to explain how the changes would be integrated into the time series so that previous years and future years of Gulf surveys remain comparable.

- [ ] Decide on how much time we should ask for in front of the Joint Plan Team in November
- [ ] Put together a package of data and analyses to justify Option 1 and Option 2 and discuss with Stan ahead of PT meeting
